### PR TITLE
Ziyuan

### PR DESCRIPTION
**Pull request by ZIYUANLIU98**

### CODE REVIEW

**Review by petelliott — CHANGES_REQUESTED** [review]
i'm okay with renaming base to barbase, but can you delete base.html and base.css?

File: users/views.py [hunk 1]
```diff
@@ -53,3 +53,20 @@ def authenticated_test(request):
...
+
+def homepage(request):
+
+    return render(request, "users/homepage.html" )
+
+def friend(request):
+
+    return render(request, "users/homepage.html" )
+
+def post(request):
```
Comment (petelliott) [inline users/views.py]: I think that blogging should be in a seperate app

**Review by petelliott — CHANGES_REQUESTED** [review]

File: users/views.py (hunk 1 above)
Comment (ZIYUANLIU98) [reply users/views.py]: Homepage has 3 buttons of  options, so I create barbase.html for convenience. I can try to use base.html in homepage.

Comment (ZIYUANLIU98) [reply users/views.py]: Got it. I'll modify it later.

Comment (petelliott) [reply users/views.py]: Thanks!

**Review by petelliott — APPROVED** [review]

**Pull request merged**